Webapp/sources/blueprints/reaction_table/reaction_classification.py

```diff
@@ -0,0 +1,148 @@
+from rdkit import Chem
```

Comment:
this file could probably go to services as well

Reply:
done